### PR TITLE
fix: console delete, delete big checksign

### PR DESCRIPTION
**Pull request by tenadopt**
Delete console.error and console.log.
Fix big sign of deleting successfully.

### CODE REVIEW

File: src/app/index.tsx [hunk 1]
```diff
@@ -5,21 +5,25 @@ import { Provider } from 'react-redux';
...
 
 if (container) {
     const root = createRoot(container);
+
     root.render(
         <QueryClientProvider client={queryClient}>
             <Provider store={store}>
                 <App />
                 <ReactQueryDevtools initialIsOpen={false} />
+                <ToastContainer autoClose={3000} hideProgressBar />
```
Comment (shev7) [inline src/app/index.tsx:23]: Why are you importing container here and in app? it should be only in one place

File: src/features/imageEditor/ui/CanvasEditor.tsx [hunk 2]
```diff
@@ -6,6 +6,8 @@ import {
 } from '@mui/material';
 import { setBrushSizeWithState, setColorWithState } from 'features/imageEditor/model/imageEditorSlice';
 import { useSaveImage } from 'features/imageGallery/api/useImageQueries';
+import { toast, ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
```
Comment (shev7) [inline src/features/imageEditor/ui/CanvasEditor.tsx]: the same here you don't need to import styles for container everywhere

**Review by shev7 — CHANGES_REQUESTED** [review]
You just need import styles once in app

File: src/features/imageEditor/ui/CanvasEditor.tsx (hunk 2 above)
Comment (shev7) [reply src/features/imageEditor/ui/CanvasEditor.tsx]: or in index file

File: src/app/index.tsx (hunk 1 above)
Comment (tenadopt) [reply src/app/index.tsx:23]: Done.
Delete <ToastContainer /> in App. Also add in index ReactToastify.css

**Pull request merged**